packages/slice-machine/sentry-next.properties

```diff
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
+defaults.org=prismicio
+defaults.project=slice-machine
```

Comment:
should we make the names more explicit ?

Reply:
It could be `Slice Machine UI` (same name as the package) or we can leave it as Slice Machine and call other apps like `Slice Machine Init`

Reply:
Updated